recommence/Checkpoint.py

```diff
@@ -1,9 +1,18 @@
 import os
 import shutil
 import pickle
+import logging
 from typing import Any, Dict, Callable, TypeVar
 
 T = TypeVar('T')
+
+logger = logging.getLogger('recommence')
+logger.setLevel(logging.DEBUG)
```

Comment:
Lines 10-14 (inclusive) we should actually leave to the consumer to manage. Basically, our responsibility here is to emit appropriate logging messages to the logger so that they are available _if_ someone wants them. Then the consumer is responsible for deciding:
1. What level of logs
2. Where they should go (to stderr, to disk, etc.)
3. What format they should take

Reply:
That makes sense. 🙆🏻‍♀️ so, I should simply get rid of lines 10-14, and let the user decide for them later by calling `logging.basicConfig()` in their own code, right?

Reply:
Yup, exactly! :+1: